leakcanary-android/src/main/res/values/leak_canary_int.xml

```diff
@@ -16,4 +16,5 @@
   -->
 <resources>
   <integer name="leak_canary_max_stored_leaks">7</integer>
+  <integer name="leak_canary_watch_delay_seconds">5</integer>
```

Comment:
I'd rather use milliseconds here, someone might want to be agressive and go for say `1500` ms.